tests/packagedcode/data/buildpack/paketo-buildpacks/dotnet-execute/buildpack.toml

```diff
@@ -0,0 +1,20 @@
+api = "0.8"
+
+[buildpack]
+  description = "A buildpack for running the `dotnet execute` command for an app"
+  homepage = "https://github.com/paketo-buildpacks/dotnet-execute"
+  id = "paketo-buildpacks/dotnet-execute"
+  keywords = ["dotnet"]
+  name = "Paketo Buildpack for .NET Execute"
```

Comment:
This is not really a package name proper, we should not populate this in the package name? 
The `namespace/name` combination seems to be present in the `id` field we should parse and populate from there.